exonum/src/node/basic.rs

```diff
@@ -18,31 +18,31 @@ use std::{error::Error, net::SocketAddr};
 
 use super::{NodeHandler, NodeRole, RequestData};
 use helpers::Height;
-use messages::{Any, Connect, Message, PeersRequest, RawMessage, Status};
+use messages::{Any, Message, PeersExchange, PeersRequest, RawMessage, Status};
+use node::ConnectInfo;
 
 impl NodeHandler {
     /// Redirects message to the corresponding `handle_...` function.
     pub fn handle_message(&mut self, raw: RawMessage) {
         match Any::from_raw(raw) {
-            Ok(Any::Connect(msg)) => self.handle_connect(msg),
             Ok(Any::Status(msg)) => self.handle_status(&msg),
             Ok(Any::Consensus(msg)) => self.handle_consensus(msg),
             Ok(Any::Request(msg)) => self.handle_request(msg),
             Ok(Any::Block(msg)) => self.handle_block(&msg),
             Ok(Any::Transaction(msg)) => self.handle_tx(msg),
             Ok(Any::TransactionsBatch(msg)) => self.handle_txs_batch(&msg),
+            Ok(Any::PeersExchange(msg)) => self.handle_peers_exchange(&msg),
```

Comment:
Maybe it's a bit late, but I think `Peers` or `PeerList` (or at least `ExchangePeers`) will be a better wording than `PeersExchange`. It would be in-line with other messages which are descriptive nouns of their content. This structure literally contains a "list of known peers". (And `PeerList` can't be misread for stupid jokes.)